climada/util/lines_polys_handler.py

```diff
@@ -951,6 +959,29 @@ def _line_to_pnts(gdf_lines, res, to_meters):
     return gdf_points
 
 
+def _line_fraction(length, res):
+    """
+    Comoute the fraction in which to divide a line of given length at given resolution
+
+    Parameters
+    ----------
+    length : float
+        Length of a string
+    res : float
+        Resolution (length of a string element)
+
+    Returns
+    -------
+    np.ndarray
+        Array of the fraction at which to divide the string of given length
+        into points at the chosen resolution.
+
+    """
+    nb_points = _pnts_per_line(length, res)
```

Comment:
`_pnts_per_line` is a single line of code that is only executed from here. I suggest moving this line here and removing the `_pnts_per_line` function

Reply:
I disagree here. This makes the code more readable and also easier to test. Getting the number of points per line is a functionally relevant step and should not be buried imho. What do you think?